### ISSUE TITLE
Syncback - Issue copying rsync binary into container after recent changes

### ISSUE THREAD

**Issue by TimothyLoyer** [tilt-dev/tilt-extensions#209]
I was working through Syncback errors with a coworker today and she noticed that this recent commit quotes some arguments in a way that breaks correct argument parsing. After reverting krsync.sh to the version int he last commit, it worked for me again.

You can see here that the last part is quoted, but should not be since it is not actually a single argument but many.
https://github.com/tilt-dev/tilt-extensions/blame/018a67d9a544f9f1df6238074db9e781ef7af60b/syncback/krsync.sh#L18

**Comment by TimothyLoyer** [comment]
I'm not convinced that #210 is going to fix the quoted $@ argument on line 18 here- I don't believe this should be closed. The script was failing much earlier than the fix starts.

**Comment by landism** [comment]
> I'm not convinced that #210 is going to fix the quoted $@ argument on line 18 here- I don't believe this should be closed. The script was failing much earlier than the fix starts.

Can you share some of what you're seeing?
1. What error output are you getting?
2. What are you seeing that leads you to believe the script fails much earlier than the fix starts? (I've sometimes had issue get this kind of debug info from this script due to the way it re-executes itself via rsync)

FWIW, here is the error that I reproduced at HEAD and fixed with #210. If your error is the same, would you mind giving the fix a try?
```
Running cmd: /Users/matt/go/src/github.com/tilt-dev/servantes/tilt_modules/syncback/krsync.sh deploy/matt-spoonerisms -aOv --progress --stats  -T=/tmp/rsync.tilt --exclude=".git***" --exclude=".gitignore***" --exclude=".dockerignore***" --exclude="Dockerfile***" --exclude=".tiltignore***" --exclude="Tiltfile***" --exclude="tilt_modules***" --exclude="package.json***" --exclude="yarn.lock***"  dummy:/app/node_modules/ spoonerisms/src/node_modules
error: pod, type/name or --filename must be specified
rsync: connection unexpectedly closed (0 bytes received so far) [Receiver]
rsync error: error in rsync protocol data stream (code 12) at io.c(228) [Receiver=3.2.3]
```

**Comment by kuzaxak** [comment]
If you specify namespace and container you will get commands like:

```
Running cmd: /Users/kkkk/projects/glia/sm-configuration/dev-ns/tilt_modules/syncback/krsync.sh "deploy/operator-ui-visual-tests -c operator-ui" -aOv --progress --stats  -T=/tmp/rsync.tilt --exclude=".git***" --exclude=".gitignore***" --exclude=".dockerignore***" --exclude="Dockerfile***" --exclude=".tiltignore***" --exclude="Tiltfile***" --exclude="tilt_modules***"  dummy@dev-v-kuznichenkov:/app/operator_ui/gemini-report/ ./operator_ui/gemini-report/
Error from server (NotFound): deployments.apps "operator-ui-visual-tests -c operator-ui" not found
cat: stdout: Broken pipe
rsync: connection unexpectedly closed (0 bytes received so far) [Receiver]
rsync error: error in rsync protocol data stream (code 12) at io.c(228) [Receiver=3.2.3]
/Users/kkkk/projects/glia/sm-configuration/dev-ns/tilt_modules/syncback/krsync.sh "deploy/operator-ui-visual-tests -c operator-ui" -aOv --progress --stats  -T=/tmp/rsync.tilt --exclude=".git***" --exclude=".gitignore***" --exclude=".dockerignore***" --exclude="Dockerfile***" --exclude=".tiltignore***" --exclude="Tiltfile***" --exclude="tilt_modules***"  dummy@dev-v-kuznichenkov:/app/operator_ui/gemini-report/ ./operator_ui/gemini-report/ exited with exit code 12
Build Failed: Command "/Users/kkkk/projects/glia/sm-configuration/dev-ns/tilt_modules/syncback/krsync.sh \"deploy/operator-ui-visual-tests -c operator-ui\" -aOv --progress --stats  -T=/tmp/rsync.tilt --exclude=\".git***\" --exclude=\".gitignore***\" --exclude=\".dockerignore***\" --exclude=\"Dockerfile***\" --exclude=\".tiltignore***\" --exclude=\"Tiltfile***\" --exclude=\"tilt_modules***\"  dummy@dev-v-kuznichenkov:/app/operator_ui/gemini-report/ ./operator_ui/gemini-report/" failed: exit status 12
```

`krsync.sh "deploy/operator-ui-visual-tests -c operator-ui"` container shouldn't be escaped together:

Error from kubernetes:

```
kubectl exec 'deploy/operator-ui-visual-tests -c operator-ui' -- ls /bin/rsync.tilt
Error from server (NotFound): deployments.apps "operator-ui-visual-tests -c operator-ui" not found
```

**Comment by nicksieger** [comment]
@kuzaxak I think I might have fixed the issue in #278, but I need to work on some tests for the syncback extension because we've had lots of issues related to it. Can you try the version there and report back?
